### DISCUSSION TITLE
External local input folder

### DISCUSSION THREAD

**Discussion by lucasnantonio** [11ty/eleventy#1941] [Q&A] [answered]
Hi! I'm new to 11ty and trying to figure out if my use case is applicable.

I keep a .md journal in a local folder synced to Dropbox, so I can edit from any device.

I want to use this external, local folder as input, without making it a child of my eleventy repo.

Here's what I'm trying to achieve:

```
/ journal
--- entry1.md
--- entry2.md

/ my-11ty-blog
--- _includes
--- _data
--- etc... 
```

I have succeeded in using the external folder as input by using an absolute path in input, like:

`~/user/myname/journal`

but this causes problems when trying to apply global layouts.

any tips on how to achieve such a structure?

**Comment by pdehaan** [comment, marked as answer]
Interesting. I've never tried an input directory that only had SOME of the files (but not layout or other files).

Wonder if you need to solve that with a symlink or git submodule.

```
/journal
--- entry1.md
--- entry2.md

/ my-11ty-blog
--- _includes/
--- _data/
--- journal/ (symlinked to the folder above)
--- .eleventy.js
--- etc
```

This way your input directory can still be `.` and is at the same location as your _includes/ and global data folder. I can't see why it _wouldn't_ work (famous last words), but it'd be a fun experiment.

**Reply by pdehaan** [reply]
Seems to work! My /journal and /my-11ty-blog folders are at the same level, and ./src/journal is symlinked to "../../journal" (I ran <kbd>ln -s ../../journal</kbd> from within my "./src/" folder). In your case it might be easier to stick w/ an absolute path to your Dropbox, or whatever works for you:

```sh
tree -I node_modules .
.
├── package-lock.json
├── package.json
├── src
│   ├── _data
│   │   └── env.js
│   ├── _includes
│   │   ├── footer.njk
│   │   └── header.njk
│   └── journal -> ../../journal
└── www
    └── journal
        ├── post-1
        │   └── index.html
        ├── post-2
        │   └── index.html
        ├── post-3
        │   └── index.html
        └── post-4
            └── index.html

10 directories, 9 files
```

And I assume if you need data directory files (if you want to apply tags or frontmatter data to each journal entry), that would have to live in your Dropbox folder as well, since I can't think of a way to have it symlinked but merge sources from two places.

**Reply by lucasnantonio** [reply]
Can confirm the symlink works. Thank you so much!
